File: Ahorn/triggers/GokuroNormalizeTrigger.jl
module Sardine7GokuroNormalizeTrigger

using ..Ahorn, Maple

@mapdef Trigger "Sardine7/GokuroNormalizeTrigger" GokuroNormalizeTrigger(x::Integer, y::Integer, width::Integer=16, height::Integer=16, count::Integer=1)

const placements = Ahorn.PlacementDict(
    "Gokuro (Normalize, Sardine7)" => Ahorn.EntityPlacement(
        GokuroNormalizeTrigger,
        "rectangle"
    )
)

end

File: Loenn/triggers/gokuro_normalize_trigger.lua
local gokuroNormalize = {}

gokuroNormalize.name = "Sardine7/GokuroNormalizeTrigger"

gokuroNormalize.fieldInformation = {
    count = {
        fieldType = "integer",
        minimumValue = 1
    }
}

gokuroNormalize.placements = {
    name = "gokuro_normalize",
    data = {
        count = 1
    }
}

return gokuroNormalize
